src/components/AccountSelector/AccountSelectButton/AccountSelector.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import { FiUser } from "react-icons/fi";
+import { MassaToken } from "../../../assets/svg-components/MassaToken";
+import { AccountSelectorButton } from "./AccountSelectorButton";
+
+export default {
+  title: "Components/Account Selector",
+  component: AccountSelectorButton,
+};
+
+const theme = "theme-light";
+
+const args = {
+  profileimage: <FiUser className="text-neutral h-6 w-6" />,
+  accountname: "account #",
+  svg: <MassaToken size={"24"} />,
+  accountbalance: "0,000.00",
```

Comment:
The backend team should be asked in what format this number is received. Is it a number? Or a string?

I don't know what is expected.

Reply:
well for now i'm guessing it's a string,  because if we want it to be an int we'll have to parse it, on the other hand we can do a .toString() to go from int to string ? @mazmassa what are your thoughts ?

Reply:
Size in string for me is weird. It should be integer indeed. 

However, if the `<Icons>` needs as string because of the SVG I would update icons **first** to receive in integer (let's parse in Icons _toString()_ then we can start using correctly as it should be: integer.

Reply:
yeah the icons has been changed !